<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app</class>
 <widget class="QMainWindow" name="app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>app</string>
  </property>
  <property name="styleSheet">
   <string notr="true">app {
  background-color: #f0f0f0;
  color: #000000;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="barra_h">
      <property name="styleSheet">
       <string notr="true">QFrame#barra_h {
    background-color: #252525;
    border: 1px solid #7a7a7a;
    color: #ffffff;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame {
    background-color: #f0f0f0;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">QFrame#frame_2 {
  background-color: #ffffff;
  border: 1px solid #c2c2c2;
  border-radius: 8px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QFrame" name="frame_5">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_5 {
  background-color: #f0f0f0;
  border: 1px solid #c2c2c2;
  border-radius: 4px;
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <property name="bottomMargin">
                    <number>4</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QLabel" name="name">
                         <property name="styleSheet">
                          <string notr="true">color: #252525;</string>
                         </property>
                         <property name="text">
                          <string>Nombre:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="nombre_label">
                         <property name="styleSheet">
                          <string notr="true">color: #252525;</string>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <widget class="QLabel" name="mime">
                         <property name="styleSheet">
                          <string notr="true">color: #252525;</string>
                         </property>
                         <property name="text">
                          <string>Formato:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="formato_label">
                         <property name="styleSheet">
                          <string notr="true">color: #252525;</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_3">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_3 {
  background-color: #f0f0f0;
  border: 1px solid #c2c2c2;
  border-radius: 4px;
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_11">
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <property name="bottomMargin">
                    <number>4</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QLabel" name="path">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                       <property name="text">
                        <string>Ruta:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="ruta_label">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_4">
                  <property name="styleSheet">
                   <string notr="true">QFrame#frame_4 {
  background-color: #f0f0f0;
  border: 1px solid #c2c2c2;
  border-radius: 4px;
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <property name="bottomMargin">
                    <number>4</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="tiempo">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                       <property name="text">
                        <string>Fecha:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="fecha_label">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="tamano">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                       <property name="text">
                        <string>Tama√±o:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="tamano_label">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="mb_size">
                       <property name="styleSheet">
                        <string notr="true">color: #252525;</string>
                       </property>
                       <property name="text">
                        <string>MB</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="encriptar">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
  color: #ffffff;
  background-color: #252525;;
  border: 1px solid #c2c2c2;
  border-radius: 4px;
  padding: 4px 18px 4px 18px;
}

QPushButton:hover {
  background-color: #000000;
  color: #f0f0f0;
}

QPushButton:pressed {
  background-color: #c2c2c2;
  color: #252525;
}

QPushButton:disabled {
  background-color: #c2c2c2;
  color: #252525;
}</string>
                </property>
                <property name="text">
                 <string>Encriptar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="desencriptar">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
  color: #252525;
  background-color: #ffffff;
  border: 1px solid #252525;
  border-radius: 4px;
  padding: 4px 8px 4px 8px;
}

QPushButton:hover {
  background-color: #000000;
  color: #f0f0f0;
}

QPushButton:pressed {
  background-color: #c2c2c2;
  color: #252525;
}

QPushButton:disabled {
  background-color: #c2c2c2;
  color: #252525;
}</string>
                </property>
                <property name="text">
                 <string>Desencriptar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="eliminar">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
  color: #ffffff;
  background-color: #d92626;
  border: none;
  border-radius: 4px;
  padding: 4px 8px 4px 8px;
}

QPushButton:hover {
  background-color: #e46464;
  color: #000000;
}

QPushButton:pressed {
   background-color: #981b1b;
   color: #000000;
}

QPushButton:disabled {
   background-color: #981b1b;
   color: #000000;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="icons.qrc">
                  <normaloff>:/basic/icons/trash-icon.svg</normaloff>:/basic/icons/trash-icon.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenu {
    color: #ffffff;
    background-color: #252525;
    border: 1px solid #c2c2c2;
}

QMenuBar {
    color: #ffffff;
    background-color: #252525;
    border: 1px solid #c2c2c2;
}

QMenuBar::item:hover {
    background-color: #7a7a7a;
}
        
QMenuBar::item:selected {
    background-color: #7a7a7a;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">QMenuBar::item:hover {
    background-color: #7a7a7a;
}</string>
    </property>
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionabout_Pentaloon"/>
    <addaction name="actionabout_third_parties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuabout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionabout_Pentaloon">
   <property name="text">
    <string>about Pentaloon</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string> Close File</string>
   </property>
  </action>
  <action name="actionabout_third_parties">
   <property name="text">
    <string>about third parties</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
